tests/test_questions.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+import os
+import json
+
+from app.views import create_app
+from app.models import Questions, Answers
+
+class StackOverflow_lite(unittest.TestCase):
+    """This class represents Questions and Answers posted."""
+
+    def setUp(self):
+        """Define test variables and initialize app."""
+        self.app = create_app(config_name="testing")
+        self.client = self.app.test_client()
+        self.questions = {'Question': 'What is a flask restful api', 'Date posted': '12th August 2018'}
+        self.answers = {'Question': 'Flask restful api', 'Date posted': '14th August 2018'}
+        
+    def test_post_question(self):
+        """Testing posting a question."""
+        response = self.client.post(
+            '/api/v1/questions', data=json.dumps(self.questions), content_type='application/json')
+        self.assertEqual(response.status_code, 201)
+
+    def test_view_all_questions(self):
+        """Test to view all questions."""
+        response = self.client.get(
+            '/api/v1/questions', content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+    def test_get_one_question(self):
+        """Test to get single question."""
+        response = self.client.get(
+            '/api/v1/questions/1', data=json.dumps(self.questions), content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+    def test_post_answer_to_question(self):
+        """Test to post answer"""
+        response = self.client.post(
+            '/api/v1/questions/1/answers', data=json.dumps(self.answers), content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+
+    def test_no_question(self):
+        """Test to no question"""
+        response = self.client.get( 
+            '/api/v1/questions/88', data=json.dumps(self.answers), content_type='application/json')  
+        self.assertEqual(response.status_code, 200)
```

Comment:
how come `test_no_question` returns a status code of `200`

Reply:
Color please

Reply:
`200` was the most appropriate since there was no error.

Reply:
read about 404 and relate to what you are trying to achieve

Reply:
I used 200 reasons being the request was processed successfully and then it gives an empty list while using 404 means the server request was not found or else the client was unable to communicate with server, in short, the request was received and successful, the 4xx codes means there was a client error whereas in this case there was none